src/components/common/Navbar/Navbar.tsx

```diff
@@ -39,6 +44,7 @@ function Navbar() {
   const [showMenu, setShowMenu] = useState(false)
   const { isMobile } = useWindowDimensions()
   const dismissTransition = useRef<Callback | undefined>()
+  const handleCloseSlideOver = () => setShowMenu(false)
```

Comment:
`useCallback` in here to avoid re-renders

Reply:
This callback isn't used inside hooks or memoized components.